tools/PI/DevHome.PI/Models/ClipboardMonitor.cs

```diff
@@ -126,29 +112,24 @@ private ClipboardContents ParseClipboardContents(string text)
             }
 
             newContents.Raw = original;
-            newContents.Hex = errorAsInt is not null ? Convert.ToString((int)errorAsInt, 16) : original;
-            newContents.Dec = errorAsInt is not null ? Convert.ToString((int)errorAsInt, 10) : original;
+            newContents.Hex = Convert.ToString((int)errorAsInt, 16);
+            newContents.Dec = Convert.ToString((int)errorAsInt, 10);
 
-            // Is there an error code on here?
-            // if (ErrorLookupHelper.ContainsErrorCode(text, out var hresult))
-            if (errorAsInt is not null)
+            var errors = ErrorLookupHelper.LookupError((int)errorAsInt);
+            if (errors is not null)
             {
-                var errors = ErrorLookupHelper.LookupError((int)errorAsInt);
-                if (errors is not null)
+                foreach (var error in errors)

```

Comment:
NIT: errors.Where(x => x is not null);

Reply:
I believe in this case it will never be null. No need to add a filter to the enumerator.